generator/src/test/java/com/scottlogic/deg/generator/cucumber/features/operators/GranularTo.feature

```diff
@@ -97,3 +97,85 @@ Feature: User can specify that a numeric value is of a decimalised value to a sp
     Given foo is granular to null
     Then the profile is invalid because "Couldn't recognise 'value' property, it must be set to a value"
     And no data is created
+
+
+  Scenario: Running granularTo against a non contradicting granularTo should be successful
+    Given foo is granular to 1
+    And foo is granular to 1
+    And foo is greater than 0
+    And the generator can generate at most 5 rows
+    Then the following data should be generated:
+      | foo |
+      | 1   |
+      | 2   |
+      | 3   |
+      | 4   |
+      | 5   |
+
+  Scenario: Running granularTo run against a non contradicting not granularTo should be successful
```

Comment:
This scenario should be ignored, not(granularTo 0.1) is undefined behaviour

Reply:
Yea marked it as ignore